.claude/hooks/pre_tool_use.py

```
@@ -0,0 +1,142 @@
+        r'\.',          # Current directory
+        r'\.\s*$',      # Current directory at end of command
+    ]
+    
+    if re.search(r'\brm\s+.*-[a-z]*r', normalized):  # If rm has recursive flag
+        for path in dangerous_paths:
+            if re.search(path, normalized):
+                return True
+    
+    return False
+
+def is_env_file_access(tool_name, tool_input):
```

Ruff (0.12.2)

Missing return type annotation for public function `is_env_file_access`

Add return type annotation: `bool`

(ANN201)